example/Gemfile

```diff
@@ -1,10 +1,10 @@
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.5.1'
+gem 'rails', '~> 4.2'
 gem 'sqlite3'
 
 gem 'dotenv-rails'
-gem 'shopify_app', '~> 7.0.0'
+gem 'shopify_app', '~> 7.2.0'
```

Comment:
I think its best to not specify the ShopifyApp version at all.. let the example run with the latest release